packages/beacon-node/src/chain/archiveStore/historicalState/worker.ts

```diff
@@ -80,6 +80,22 @@ if (metricsRegister) {
       name: "lodestar_historical_state_stfn_num_effective_balance_updates_count",
       help: "Count of effective balance updates in epoch transition",
     }),
+    validatorsInQueueToActive: metricsRegister.gauge({
+      name: "lodestar_historical_state_stfn_validators_in_queue_to_active_total",
+      help: "Total count of validators in the queue to active",
```

Comment:
this reads wrong to me, did you mean activate?

```suggestion
      help: "Total count of validators in the queue to activate",
```

(same renaming needed on dashboard and variable)

Reply:
How about `Total count of validators in the activation queue` and `Total count of validators in the exit queue`? Sounds better for me.